modules/helm/helm.mk

```diff
@@ -74,8 +76,11 @@ $(helm_chart_archive): $(helm_chart_sources) | $(NEEDS_HELM) $(NEEDS_YQ) $(bin_d
 ## Will also create a non-v-prefixed tag for the OCI image.
 ## @category [shared] Publish
 helm-chart-oci-push: $(helm_chart_archive) | $(NEEDS_HELM) $(NEEDS_CRANE)
-	$(HELM) push "$(helm_chart_archive)" "oci://$(helm_chart_image_registry)"
-	$(CRANE) copy "$(helm_chart_image_name):$(helm_chart_image_tag)" "$(helm_chart_image_name):$(helm_chart_image_tag:v%=%)"
+	$(HELM) push "$(helm_chart_archive)" "oci://$(helm_chart_image_registry)" 2>&1 \
+		| grep -o "sha256:.\+" \
+		| tee $(helm_digest_path)
+
+	$(CRANE) copy "$(helm_chart_image_name)@$(call helm_digest)" "$(helm_chart_image_name):$(helm_chart_image_tag:v%=%)"
```

Comment:
suggestion (non-blocking): Not required to merge (it was there before this PR) but there are lot of magic incantations in various modules which would benefit from more comments. Documenting what `$(helm_chart_image_tag:v%=%)` does would help a lot for people less familiar with make / bash!

Reply:
The PR already merged, but I'll explain it here:
`$(helm_chart_image_tag:v%=%)` removes the `v` prefix from the value stored in `helm_chart_image_tag`.
See https://www.gnu.org/software/make/manual/html_node/Substitution-Refs.html for the manual on the syntax.